src/OrchardCore/OrchardCore.ContentLocalization.Abstractions/OrchardHelperExtensions.cs

```diff
@@ -15,11 +15,12 @@ public static class OrchardHelperExtensions
         /// </summary>
         /// <param name="orchardHelper">The <see cref="IOrchardHelper"/>.</param>
         /// <param name="contentItem">The <see cref="ContentItem"/> in which to get its culture.</param>
+        /// <param name="defaultCulture">The <see cref="CultureInfo"/> the default culture it should fallback to.</param>
         /// <returns></returns>
-        public async static Task<CultureInfo> GetContentCultureAsync(this IOrchardHelper orchardHelper, ContentItem contentItem)
+        public async static Task<CultureInfo> GetContentCultureAsync(this IOrchardHelper orchardHelper, ContentItem contentItem, CultureInfo defaultCulture = CultureInfo.CurrentUICulture)
```

Comment:
Here I'm adding an optional defaultCulture param if anyone needs to do the same kind of thing somewhere else where the ContentManager is not injected e.g. a view template.